.devops/templates/build-test-lint.yml

```diff
@@ -18,8 +18,8 @@ steps:
       # @fluentui/api-docs is used within apps/public-docsite-resources/just.config.ts,
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32108&commitId=93e247468c14453c8044ac34e623d9283d4528f5&env=prod&clientType=FLUENTUI)


  <details><summary><b>Avatar Converged</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|5| Changed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|0| Removed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>